keras/layers/convolutional.py

```diff
@@ -1411,9 +1411,9 @@ class SeparableConv1D(_SeparableConv):
             one of `"channels_last"` or `"channels_first"`.
             The ordering of the dimensions in the inputs.
             `"channels_last"` corresponds to inputs with shape
-            `(batch, height, width, channels)` while `"channels_first"`
+            `(batch, length, channels)` while `"channels_first"`
```

Comment:
Would be better to be consistent with Conv1D which uses `steps`
https://github.com/fuzzythecat/keras/blob/398171294c465e6d8b60727ea77efae44609529a/keras/layers/convolutional.py#L305